package.json

```
@@ -8,6 +8,7 @@
       "libs/*",
       "libs/@hashintel/*",
       "libs/@local/*",
+      "libs/@local/*/typescript",
```

We want the python package to be part of the workspace as well.
```suggestion
      "libs/@local/*/typescript",
      "libs/@local/*/python",
```